biblio-iacr.el

```diff
@@ -0,0 +1,91 @@
+;;; biblio-iacr.el --- description -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2020 Lorenzo Martinico
+
+;; Author: Lorenzo Martinico <lzmartinico@gmail.com>
+;; URL: https://github.com/cpitclaudel/biblio.el
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+;;
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+;;
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+;;; Commentary:
+;;
+;; This file implements a backend for the for the `biblio' package (which see for more
+;; documentation).
+;;
+;;; Code:
+
+(require 'biblio-core)
+
+(defun biblio-iacr--forward-bibtex (metadata forward-to)
+  "Forward BibTeX for IACR entry METADATA to FORWARD-TO."
+    (funcall forward-to
+         (format "@misc{cryptoeprint:%s,\n    author = {%s},\n    title = {%s},\n    howpublished = {%s},\n    year = {%s},\n    note = {\\url{%s}},\n}"
+                 (biblio-alist-get 'eprintid metadata)
+                 (car (biblio-alist-get 'authors metadata))
```

Comment:
Do you really want to keep just one author?

Reply:
Unfortunately IACR only returns an AND separated string containing all the authors. Maybe I could split it into a list?

Reply:
I'd split on `" and "`, yes, and reassemble in the bibtex generator,